server/controllers/Conversations.js

```diff
@@ -227,22 +222,20 @@ module.exports.leaveConversation = async (req, res) => {
             user_id : user_id,
             conversation_id: conversation_id
         }
-    })
-    //catch any errors
-    .catch(err => {
+    }).catch(err => { //catch any errors
         res.status(500).send({
             msg:
             err.message || "Some error occurred while removing the user from the conversation."
         });
     });
 
     if(!deleted_participant_row){
-        return res.send({msg:"User couldn't be removed from conversation. Probably wasn't a participant."})
+        return res.send({msg:"User couldn't be removed from conversation. Probably wasn't a participant."});
```

Comment:
Maybe we should return an error status here and on line 185

Reply:
Is a 200 code ok here?

Reply:
Yh. I removed the comment, because this could be dealt with in the api overhaul. But you can update the status code here and on line 185 if you want.